packages/cli/internal/pkg/cli/workflow/workflow_run.go

```diff
@@ -20,6 +20,7 @@ func (m *Manager) RunWorkflow(contextName, workflowName, inputsFileUrl string, o
 	}
 	m.calculateFinalLocation()
 	m.readInput(inputsFileUrl)
+	m.writeTempManifest()
 	m.uploadInputsToS3()
 	m.parseInputToArguments()
```

Comment:
1. refactor these steps ahead of `isUploadRequired`. 
2. Because there is already a temp folder created, you should just add the input read from `--inputsFile` into the MANIFEST.json.
3. Remove the `uploadInputsToS3` because the temp MANIFEST.json contains all workflow paths that we need.